Procfile

```diff
@@ -1,3 +1,3 @@
 release: bundle exec rails db:migrate
 web: bundle exec puma -C config/puma.rb
-worker: bundle exec sidekiq -c 15
+worker: bundle exec sidekiq -c ${SIDEKIQ_CONCURRENCY}
```

Comment:
Is this change supposed to be part of this PR? I'd probably do any sidekiq config changes separately